tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/constant/SimpleConstantNegativeTest.java

```diff
@@ -147,7 +147,11 @@ public void testNegative() {
                 294, 7);
         BAssertUtil.validateError(compileResult, index++, "expression is not a constant expression",
                 294, 17);
-        BAssertUtil.validateError(compileResult, index, "constant declarations are allowed only at module level",
+        BAssertUtil.validateError(compileResult, index++, "constant declarations are allowed only at module level",
                 297, 5);
+        BAssertUtil.validateError(compileResult, index++, "type is required for constants with expressions",
+                301, 25);
+        BAssertUtil.validateError(compileResult, index,
+                "cannot resolve constant 'UT_COUNT'", 302, 27);
```

Comment:
Can't these be merged?

```suggestion
        BAssertUtil.validateError(compileResult, index, "cannot resolve constant 'UT_COUNT'", 302, 27);
```